<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAuthor</class>
 <widget class="QWidget" name="FormAuthor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить автора</string>
  </property>
  <widget class="QPushButton" name="pushAuthor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>327</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineID">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Страна</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCountry">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Годы жизни (в формате Год рождения:Год смерти)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineYears">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
